Dockerfile

```diff
@@ -127,6 +127,10 @@ RUN /root/.bitrise/tools/stepman update
 
 COPY ./ssh/config /root/.ssh/config
 
+# ------------------------------------------------------
+# --- Install Rubygems
+
+RUN gem update --system --no-document
```

Comment:
I'm not sure if this is required, I believe this is part of ruby's `make && make install` (https://github.com/bitrise-docker/bitrise-base/pull/75/files#diff-3254677a7917c6c01f55212f86c57fbfR56) to get the latest rubygems.